### PR TITLE
Json support

### PR DESCRIPTION
**Pull request by svandenb-dev**

### CODE REVIEW

File: _unittest/test_00_EDB.py [hunk 1]
```diff
@@ -938,166 +938,6 @@ def test_82d_place_on_bottom_of_lam_with_mold_solder(self):
             chipEdb.close_edb()
             laminateEdb.close_edb()
 
-    def test_82e_place_zoffset_chip(self):
```
Comment (maxcapodi78) [inline _unittest/test_00_EDB.py:941]: Why did you remove the Test?

**Review by maxcapodi78 — CHANGES_REQUESTED** [review]
Check why some tests have been removed

File: _unittest/test_00_EDB.py (hunk 1 above)
Comment (svandenb-dev) [reply _unittest/test_00_EDB.py:941]: @maxcapodi78 I didn't touch anything. I got conflict and merged the main to resolve

**Review by maxcapodi78 — APPROVED** [review]
LGTM
